src/condor/submit.py

```diff
@@ -52,11 +54,11 @@ def main(args):
 
     # copy processor version to local directory
     if args.processor == "trigger_boosted":
```

Comment:
Wait, I missed this - why do we have to copy the processors?

Reply:
we don't need to but I was trying to have a way to remember what the jobs belonged to.. I guess we should just make tags :D - can you remove this?

Reply:
removed it